src/fiat-connect-client.ts

```diff
@@ -190,71 +122,39 @@ export class FiatConnectClientImpl implements FiatConnectApiClient {
     }
   }
 
-  /**
-   * https://en.wikipedia.org/wiki/Network_Time_Protocol#Clock_synchronization_algorithm
-   *
-   * Returns the calculated difference between the client and server clocks as a number of milliseconds.
-   * Positive values mean the server's clock is ahead of the client's.
-   * Also returns the maximum error of the calculated difference.
-   */
-  _calculateClockDiff({ t0, t1, t2, t3 }: ClockDiffParams): ClockDiffResult {
-    return {
-      diff: Math.floor((t1 - t0 + (t2 - t3)) / 2),
-      maxError: Math.floor((t3 - t0) / 2),
-    }
-  }
-
   /**
    * Returns an approximation of the current server time, taking into account clock differences
    * between client and server. Returns the earliest possible server time based on the max error
    * of the clock diff between client and server, to ensure that sessions created using this time
    * are not issued in the future with respect to the server clock.
    */
   async getServerTimeApprox(): Promise<Result<Date, ResponseError>> {
-    const clockDiffResponse = await this.getClockDiffApprox()
-    if (!clockDiffResponse.isOk) {
-      return Result.err(clockDiffResponse.error)
+    try {
```

Comment:
within this codebase, this is only used by `login`, which has logic that is more specific to SIWE clients than FiatConnect clients. How about moving the getServerTimeApprox logic to the SIWE client too?

Reply:
(before taking my suggestion, it's probably worth looking at consumers of the SDK and making sure they don't use it)

Reply:
the clock related methods in fc-client are just wrappers around the siwe client methods. I just kept them for backwards compatibility. The wallet (which I believe is the only consumer) doesn't use any of the clock methods directly. So I even considered removing these from the FiatConnectApiClient / SiweApiClient interfaces. Any other consumers we need to check? cc @jophish

Reply:
We currently don't use `getServerTimeApprox` or `getClockDiffApprox` outside of the FC SDK, but it's possible that a client may want to use them in order to present accurate expirations for quotes to an end user. For that reason, it's probably not a bad idea to continue to expose them via the FiatConnectApiClient interface as a convenience, and in turn, the `SiweApiClient` interface.

Reply:
fiatconnect validate is another consumer to check

Reply:
up to you whether you want to remove it or not